### ISSUE TITLE
[BUG] Dying with ammo count causes log to fill with errors

### ISSUE THREAD

**Issue by Grantapher** [Grantapher/ValheimPlus#30]
- **OS:** ?
- **Valheim Game Version:** 0.217.24
- **V+ Mod Version:** 0.9.10.0

**Describe the bug**

Dying with `displayBowAmmoCounts = 1` and a bow equipped causes many error logs in console.

```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
ValheimPlus.UI.HotkeyBar_UpdateIcons_Patch.DisplayAmmoCountsUnderBowHotbarIcon (HotkeyBar __instance, Player player) (at <6488f67f67354da78e129795dfd79295>:0)
ValheimPlus.UI.HotkeyBar_UpdateIcons_Patch.Postfix (HotkeyBar& __instance, Player& player) (at <6488f67f67354da78e129795dfd79295>:0)
(wrapper dynamic-method) HotkeyBar.DMD<HotkeyBar::UpdateIcons>(HotkeyBar,Player)
HotkeyBar.Update () (at <885fde5117a141d19eed8c0292c3d1fd>:0)
```

**Comment by Grantapher** [comment]
Occurs only during the loading screen after dying. Also reproducible with `displayBowAmmoCounts = 2` with no bow equipped.

```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
ValheimPlus.UI.HotkeyBar_UpdateIcons_Patch.DisplayAmmoCountsUnderBowHotbarIcon (HotkeyBar __instance, Player player) (at C:/Users/Grant/Workspace/ValheimPlus/ValheimPlus/UI/HotkeyBar.cs:34)
ValheimPlus.UI.HotkeyBar_UpdateIcons_Patch.Postfix (HotkeyBar& __instance, Player& player) (at C:/Users/Grant/Workspace/ValheimPlus/ValheimPlus/UI/HotkeyBar.cs:21)
(wrapper dynamic-method) HotkeyBar.DMD<HotkeyBar::UpdateIcons>(HotkeyBar,Player)
HotkeyBar.Update () (at <885fde5117a141d19eed8c0292c3d1fd>:0)
```

**Comment by Grantapher** [comment]
Released into [0.9.10.1-alpha01](https://github.com/Grantapher/ValheimPlus/releases/tag/0.9.10.1-alpha01).

**Comment by Grantapher** [comment]
More sources of errors released into [0.9.10.1-alpha02](https://github.com/Grantapher/ValheimPlus/releases/tag/0.9.10.1-alpha02).

**Comment by Grantapher** [comment]
Released into [`0.9.11.0`](https://github.com/Grantapher/ValheimPlus/releases/tag/0.9.11.0).
